site/app/templates/EditPronounsForm.twig

```diff
@@ -38,5 +38,5 @@
 {% endblock %}
 {% block buttons %}
     {{ block('close_button') }}
-    <input class="btn btn-primary" type="submit" value="Submit" />
+    <input class="btn btn-primary" type="submit" value="Submit" id="submit" />
```

Comment:
An ID of "submit" is overly general and is likely to conflict with other fields elsewhere on the page.  Please use a more specific identifier.

Reply:
Yeah I agree, especially since even in the cypress tests you made you are using two different submit buttons. Perhaps you could rename this one submit-pronoun-changes instead or something and the other could be submit-manage-student-columns.

Reply:
done